packages/network/src/connection/WebRtcEndpoint.ts

```diff
@@ -117,6 +120,29 @@ export class WebRtcEndpoint extends EventEmitter implements IWebRtcEndpoint {
             .addQueriedMetric('messageQueueSize', () => {
                 return Object.values(this.connections).reduce((total, c) => total + c.getQueueSize(), 0)
             })
+
+        this.statusReportIntervalMs = 5 * 60 * 1000
+        this.startConnectionStatusReport()
+    }
+
+    private startConnectionStatusReport(): void {
+        if (this.statusReportTimeout) {
+            clearTimeout(this.statusReportTimeout)
+        }
+        this.statusReportTimeout = setTimeout(() => {
```

Comment:
Maybe `setInterval` instead of `setTimeout`, to keep this as simple as possible?